test/utils_test.exs

```diff
@@ -81,12 +81,13 @@ defmodule UtilsTest do
         assert String.contains?(result, "pong")
         assert {result, 0} = System.cmd(bin_path, ["stop"])
         assert String.contains?(result, "ok")
+        # Check generated running-config
         sys_config_path = Path.join([File.cwd!, "rel", "test", "running-config", "sys.config"])
         {res, sysconfig_content} = :file.consult(to_char_list(sys_config_path))
         assert :ok = res
-        some_val = Keyword.get(List.first(sysconfig_content), :test) |> Keyword.get(:some_val)
-        assert 101 = some_val
-      #end)
+        foo = Keyword.get(List.first(sysconfig_content), :test) |> Keyword.get(:foo)
```

Comment:
I also had this checking `some_val` because it's testing part of conform's behaviour which is a more comprehensive test of the built release. Was there a reason you changed this?